### DISCUSSION TITLE
nestjs-config

### DISCUSSION THREAD

**Discussion by giscus** [tomwray13/tomray.dev#49] [Blog Comments]
# nestjs-config

Learn how to use environment variables (and other configuration values) in your NestJS projects with the Config Module.

https://www.tomray.dev/nestjs-config

**Comment by sapabg** [comment]
How do you choose witch configuration file to use? How do you run production with a different env file?

**Comment by bbangjooo** [comment]
sorry for comment unrelated to the post, but I love so much your blog design. Can I fork your blog repository and customize by myself? and If you don't mind, I want to follow you on any sns!
